solidity/test/UpdatableConsumer_test.js

```diff
@@ -22,7 +24,7 @@ contract('UpdatableConsumer', () => {
   const tokenSubnode = namehash.hash(`${tokenSubdomain}.${domain}.${tld}`)
   const oracleSubdomain = 'oracle'
   const oracleSubnode = namehash.hash(`${oracleSubdomain}.${domain}.${tld}`)
-  const specId = web3.sha3('someSpecID')
+  const specId = web3.utils.sha3('someSpecID')
```

Comment:
Would also be good to introduce a sha3 method, although, although I'd call it something like `keccak` to be more accurate. I think that's the direction solidity is moving in too.